packages/circuit-ui/components/Card/Card.tsx

```diff
@@ -13,22 +13,16 @@
  * limitations under the License.
  */
 
-import { FC, Ref, HTMLAttributes, PropsWithChildren } from 'react';
 import isPropValid from '@emotion/is-prop-valid';
 import { css } from '@emotion/react';
 
 import styled, { StyleProps } from '../../styles/styled';
 
-export interface CardProps extends HTMLAttributes<HTMLDivElement> {
+export interface CardProps {
   /**
    * The padding of the Card.
    */
   spacing?: 'mega' | 'giga';
-  /**
-   * The ref to the HTML DOM element.
-   */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ref?: Ref<any>;
```

Comment:
@connor-baer do you know if this is also okay? I assume that the styled component that we export accepts a ref already, right?

Reply:
Yes.